pilot/pkg/config/clusterregistry/testdata/cluster.yaml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Please add public/private keys to the test :-)

Reply:
Can you clarify about the public/private keys in your comments? 
This file is actually not needed anymore. Will remove it from the next commit.

Reply:
For the test data, we implemented building all the temporary files from templates and various params as data.  For the accessConfig (kubeconfig) referred to by the generated cluster-configs the conversion.go validation just validates that the files are present so those files get generated as empty files.

Reply:
For CICD/automated tests we'll need to start 2 apiserver, and have a file pointing to them. 

I think we can checkin the root certs and everything else (auth did the same), this can also serve as an example.